src/fides/api/task/execute_request_tasks.py

```diff
@@ -368,8 +362,24 @@ def run_erasure_node(
                         request_task.get_data_for_erasures() or []
                     )
 
-                    # Run the main erasure function!
-                    graph_task.erasure_request(retrieved_data)
+                    upstream_access_data: List[List[Row]] = []
+
+                    try:
+                        upstream_access_data = (
+                            get_upstream_access_data_for_erasure_task(
+                                request_task, session, resources
+                            )
+                        )
+                    except Exception as e:
```

Comment:
I'm not sure if this is off or just my understanding is off - but it seems like if its an actual error condition we should set an error state on the task and stop processing right here. If its expected that we might run into an exception or the data isn't needed (non BigQuery deletes etc), we might be able to handle it without an exception. I'm not familiar enough to know what the case is where we would get an exception and continue on to erasure_request